### PR TITLE
Model Survival and Disease control

### PR DESCRIPTION
**Pull request by stefanpfundtner**

### CODE REVIEW

**Comment by stefanpfundtner** [comment]
inally used Vadim's suggestion on using SNOMED post-coordinated expressions to model "Death due to carcinoma of the breast (disorder)" (419620001 |Death| :42752001|due to| = 254838004 |Carcinoma of breast (disorder)|)
Use this as the code of an observation and use the value to indicate if this is true (yes), false (no) or unknown.

File: input/fsh/Profiles clinical datapoints/Condition.fsh [hunk 1]
```diff
@@ -42,4 +48,54 @@ Description: "Mapping of the primary breast cancer condition to the ICHOM breast
...
+
 
+Profile: RecurrenceMethod
+Parent: Observation 
+Id: recr-method
+Title: "Recurrence Method"
+Description: "The method of confirming recurrence breast cancer"
+* insert PublicationProfileRuleset
+* code = SCT#103693007 "Diagnostic procedure"
+* value[x] only string
```
Comment (vadi2) [inline input/fsh/Profiles clinical datapoints/Condition.fsh:85]: This one should be a valueset right?

File: input/fsh/Profiles clinical datapoints/Condition.fsh [hunk 2]
```diff
@@ -28,6 +33,7 @@ Description: "Mapping of the primary breast cancer condition to the ICHOM breast
 * -> "First breast cancer"
 * bodySite -> "Laterality"
 * clinicalStatus -> "Recurrence" 
+* onset[x] -> "recurdatecancer"
```
Comment (vadi2) [inline input/fsh/Profiles clinical datapoints/Condition.fsh:36]:
```suggestion
* onset[x] -> "Date of cancer recurrence"
```

File: input/fsh/Profiles clinical datapoints/BreastCancerPatient.fsh [hunk 3]
```diff
@@ -28,6 +29,20 @@ Description: "The characteristics and demographic factors of an example patient.
...
+Description: "The characteristics and demographic factors of an example patient. The aim is to align with UScore or mCODE here in the future."
+* birthDate = "1933"
+* name.family = "Cornetet"
+* address.country = CountryCS#GB
+* gender = GenderCS#male
+* extension[educationLevel].valueCodeableConcept  = SCT#224297003
+* extension[ethnicity].valueCodeableConcept = EthnicityCS#2186-5
+* extension[race].valueCodeableConcept  = RaceCS#2054-5
+* maritalStatus = RelationshipStatusCS#D "Divorced"
+* deceasedDateTime = "1990-05-26"
```
Comment (vadi2) [inline input/fsh/Profiles clinical datapoints/BreastCancerPatient.fsh:44]: Can deceased[x] for the boolean "Vital status" also be added to the mapping? Does it work?

File: input/fsh/Terminology.fsh [hunk 4]
```diff
@@ -390,14 +390,14 @@ Description: "Valueset of the Functional Assessment of Cancer Therapy questionna
...
-// * SCT#67151002 "Histological diagnosis"
-// both --> option to use postcoordinated terms
-// * NullFlavor#UNK "unknown"
+ValueSet: RecurrenceMethod
+Id: RecurrenceMethod
+Title: "Method of confirming recurrence"
+Description: "Valueset of the methods used to confirm recurrence of breast cancer"
+* SCT#394914008 "Radiological diagnosis"
+* SCT#67151002 "Histological diagnosis"
+//both --> option to use postcoordinated terms
```
Comment (vadi2) [inline input/fsh/Terminology.fsh:399]: ICHOM hasn't provided a postcoordinated term here unfortunately - so let's define a codesystem just for this variable and use the number code. Here's an example - https://github.com/HL7/fhir-ichom-breast-cancer-ig/pull/22/files#diff-7b411e464cdc7b4cc2847257f3d596aa64d494b149a99ce2b628997c080c846aR428

**Review by vadi2 — APPROVED** [review]

File: input/fsh/Profiles clinical datapoints/BreastCancerPatient.fsh [hunk 5]
```diff
@@ -14,6 +14,7 @@ Description: "Represents the patient characteristics and demographic factors"
 * extension contains EducationLevel named educationLevel 0..1 MS
 * extension contains Ethnicity named ethnicity 0..1 MS
 * extension contains Race named race 0..1 MS
+// * extension contains DeathBC named deathbc 0..1 MS
```
Comment (vadi2) [inline input/fsh/Profiles clinical datapoints/BreastCancerPatient.fsh:17]:
```suggestion
```

**Pull request merged**